.github/workflows/build.yml

```diff
@@ -0,0 +1,20 @@
+name: CI
+on:
+  release:
+    types: [released, prereleased]
+  workflow_dispatch:  # allow manually running from the Actions tab
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      BUILDX_PLATFORMS: linux/amd64,linux/arm64,linux/arm/v7
```

Comment:
this env var can be customised also in local env: all available archs will be printed by QEMU setup